osqp_vendor/CMakeLists.txt

```diff
@@ -33,12 +33,12 @@ macro(build_osqp)
     DESTINATION
       ${CMAKE_INSTALL_PREFIX}
     USE_SOURCE_PERMISSIONS
-    PATTERN config.h EXCLUDE
   )
```

Comment:
Why this change?

Reply:
`config.h` is generated at build time, doesn't seem to be the case with `osqp_vendor`, but that's the rationale for excluding it here and why packages don't have it in their source tree.